src/main/kotlin/org/rust/lang/RustSdkType.kt

```diff
@@ -24,9 +24,15 @@ class RustSdkType : SdkType("Rust SDK") {
     override fun suggestHomePath(): String? {
         if (SystemInfo.isUnix) {
             // Check whether there is multi-rust installation
-            val multi = File(FileUtil.expandUserHome("~/.multirust/toolchains"))
-            if (multi.exists())
-                return multi.absolutePath
+            val root = File(FileUtil.expandUserHome("~/.multirust/toolchains"))
+            if (root.exists()) {
+                listOf("stable", "beta", "nightly").forEach {
+                    val multiSub = File(root, it)
+                    if (multiSub.exists()) {
+                        return multiSub.absolutePath
+                    }
+                }
```

Comment:
there is actually an issue with this: if the user has only real version numbers installed the detection will not return the parent folder anymore.

in other words: if `toolchains` only contains `1.1.0, 1.2.0, 1.3.0` it previously still returned the `toolchains` folder, but now it will move on to the next detection possibility.